pkg/controller/stacks/stack/stack.go

```diff
@@ -160,29 +184,233 @@ func (h *stackHandler) update(ctx context.Context) (reconcile.Result, error) {
 	return reconcile.Result{}, nil
 }
 
-func (h *stackHandler) createNamespacedRoleBinding(ctx context.Context, owner metav1.OwnerReference) error {
-	cr := &rbac.Role{
+func copyLabels(labels map[string]string) map[string]string {
+	labelsCopy := map[string]string{}
+	for k, v := range labels {
+		labelsCopy[k] = v
+	}
+	return labelsCopy
+}
+
+func (h *stackHandler) crdListsDiffer(crds []apiextensions.CustomResourceDefinition) bool {
+	return len(crds) != len(h.ext.Spec.CRDs)
+}
+
+// crdsFromStack fetches the CRDs of the Stack using the shared parent labels
+// TODO(displague) change this to use GET on each, the CRDs may not be ready
```

Comment:
This TODO is arguable now.  

I think the List approach and length comparison is infallible outside of admin intervention (by setting false parent labels).

Once the expected count of CRDs are accounted for, the stack can't have a discrepancy in the matched CRDs and listed CRDs because there is no inplace upgrading of Stacks (and their CRDs).

When we revisit the scenario of Foo Stack having instances residing in multiple namespaces, parent labels won't be sufficient.  We'll have to change this behavior or the approach to StackInstall/Stacks at that time.

If an expected CRD somehow survived the previous instance of the stack, the labels won't match, so the Stack won't sync (expected) until the CRD or alternate copy of the Stack is removed.

The TODO can stand and be removed when we revisit this behavior and solve the multiple namespace stack installation problem.